pkg/ui/aim/embed/build.sh

```diff
@@ -1,25 +1,24 @@
 #!/bin/sh -e
 
-# Save current directory
+# current directory and checkout location
 current=$(dirname $(realpath $0))
+repo="${current}/repo"
 
-# Create temporary directory
-repo=$(mktemp -d)
-trap "rm -rf ${repo}" EXIT
+# Checkout source and build if necessary
+if [ ! -d "${repo}" ]; then
```

Comment:
This is the meat of this change -- don't rebuild the UI if the repo dir is already cloned. Makes the dev loop faster.